src/projectes/dashboard.css

```diff
@@ -0,0 +1,109 @@
+@import url("observablehq:default.css");
+@import url("observablehq:theme-air.css");
+
+:root {
+  --sans-serif: "neue-haas-grotesk-display", sans-serif;
+  --serif: merriweather, serif;
+  --semafor-blau:#577af7;
+  --semafor-verd:#7aa047;
+  --semafor-groc:#f7d05d;
+  --semafor-taronja:#ef8733;
+  --semafor-rosa:#f09793;
+  --semafor-vermell:#b02418;
+  --dashboard-foreground: #000;
+  --dashboard-foreground-muted: #606060;
+  --dashboard-background: #f2f2f2;
+  --dashboard-background-alt: #fff;
+
+  --theme-background: #f2f2f2;
+  --theme-background-alt: #eaeaea;
+  --theme-foreground-muted: #767676;
+  --theme-background-faint: #dcdcdc;
+  --theme-foreground-focus: #3f62aa;
+  --theme-foreground-cards: #C0C6CD;
+
+  /* 
+  --theme-background: #f3f3f3;
+  --theme-background-alt: #f3f3f3;
```

Comment:
Remove the comments if they are not in use.

Reply:
Yup. Understand. But we might use that color scheme for a feature we're working on and that other collaborators may want to expand on. (Because it's project-specific, it didn't make sense to have it in the overall documentation.)